### ISSUE TITLE
Export to Excel omits values for numeric columns

### ISSUE THREAD

**Issue by landofjoe** [ag-grid/ag-grid#1487]
**I'm submitting a ...**  (check one with "x")
```
[X ] bug report => search github for a similar issue or PR before submitting
```

**Current behavior**
The demo in https://www.ag-grid.com/javascript-grid-excel/exampleExcel.html omits the values for numeric columns, including age, year, gold, silver, bronze, and total.

See attached file.
[ag-grid-export-demo.zip](https://github.com/ceolter/ag-grid/files/794951/ag-grid-export-demo.zip)

**Expected behavior**
All column values should be exported.

**Minimal reproduction of the problem with instructions**
Visit this link and click the Export to Excel button.  Examine the resulting file.

**Please tell us about your environment:**
Windows 10, Chrome & Microsoft Edge

* **ag-Grid version:** [ag-grid doc demo link]

* **Browser:**
- mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36
- Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.79 Safari/537.36 Edge/14.14393

* **Language:** [ag-grid doc demo link]

**Comment by seanlandsman** [comment]
Thanks for reporting this - we've raised an issue our side for this: AG-269

You can track its progress here: https://www.ag-grid.com/ag-grid-pipeline/ but we will update this thread too once resolved

**Comment by makinggoodsoftware** [comment]
The fix is available in 8.1.1
